CMakePresets.json

```diff
@@ -1,21 +1,30 @@
 {
-    "version": 3,

```

Comment:
Don't randomly reindent entire files. It adds unnecessary churn that makes traversing git history and merges difficult.